mp/src/materialsystem/stdshaders/wireframe_dx9.cpp

```diff
@@ -12,6 +12,9 @@
 
 DEFINE_FALLBACK_SHADER( Wireframe, Wireframe_DX9 )
 
+// Missing shaders fall back to Wireframe_DX8, so make sure that exists
+DEFINE_FALLBACK_SHADER( Wireframe_DX8, Wireframe )
```

Comment:
Given how the normal "Wireframe" shader is defined, should this just fallback to `Wireframe_DX9` as well?

Reply:
Doesn't make much of a difference, both point to the same shader. I chose `Wireframe` over `Wireframe_DX9` since it's one less thing to worry about if `Wireframe_DX9` ever changes name (e.g. to `Wireframe_DX11`).